WS2012R2/lisa/xml/STOR_VSS_Backup_Tests.xml

```diff
@@ -272,6 +273,30 @@
             <timeout>3000</timeout>
             <OnError>Continue</OnError>
         </test>
+
+        <test>
+            <testName>STOR_VSS_Backup_Disable_Enable_VSS</testName>
+            <setupScript>setupscripts\RevertSnapshot.ps1</setupScript>
+            <files>remote-scripts/ica/utils.sh</files>
+            <testScript>setupscripts\STOR_VSS_Backup_Disable_Enable_VSS.ps1</testScript>
+            <testParams>
+                <param>TC_COVERED=VSS-21</param>
+            </testParams>
+            <timeout>2400</timeout>
+            <OnError>Continue</OnError>
+        </test>
+
+        <test>
```

Comment:
Can you please combine these 2 sections?
There should be only one test case that handles the disable/enable backup integration service, and ideally only one PowerShell script that handles all operations.
Thank you!